modules/setting-higher-pid-limit-on-existing-cluster.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-configuring-pid-limits.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="setting-higher-pid-limit-on-existing-cluster_{context}"]
+= Setting a higher PID limit on an existing {product-title} cluster
+
+You can set a higher `podPidsLimit` on an existing {product-title} cluster by creating or editing a kubeletconfig that changes the `--pod-pids-limit` parameter.
+
+[IMPORTANT]
+====
+Changing the `podPidsLimit` on an existing cluster will trigger non-control plane nodes in the cluster to reboot one at a time. Red Hat recommends making this change outside of peak usage hours for your cluster, and avoiding upgrading or hibernating your cluster until all nodes have rebooted.
+====
+
+.Prerequisites
+
+* You have logged in to your Red Hat account by using the ROSA CLI.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+* Create or edit the kubeletconfig to change the PID limit.
++
+--
+** If this is the first time you are changing the default PID limit, create the kubeletconfig and set the `--pod-pids-limit` value. 
++
+[source,terminal]
+----
+$ rosa create kubeletconfig -c <cluster_name> --pod-pids-limit=<value>
+----
++
+For example, to set a maximum of 16,384 PIDs per pod for cluster my-cluster when you have never changed the default value:
```

Comment:
```suggestion
For example, to set a maximum of 16,384 PIDs per pod for cluster my-cluster when you have never created a kubeletconfig
```

Reply:
```suggestion
For example, to set a maximum of 16,384 PIDs per pod for a cluster named `my-cluster` when you have never changed the default value, run the following command:
```
Revise this introductory phrase to be a complete sentence and to introduce the command using the [OCP docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command).